### DISCUSSION TITLE
Could not access environment variable in Application.properties

### DISCUSSION THREAD

**Discussion by JerrySP** [phax/phase4#128] [General]
I have tried setting environment variables

```
$ export KEYSTORE_PSWRD=foo
# ./phase4-peppol-server-webapp
$ mvn jetty:run
```

I expect that environment variables would be successfully loaded in the following application.properties.

```
org.apache.wss4j.crypto.merlin.keystore.file=keys/ACCESS_POINT_CERTIFICATE.p12
org.apache.wss4j.crypto.merlin.keystore.password=${KEYSTORE_PSWRD}
```

But , the result is not as expected.
Do you have any idea on this?

**Comment by phax** [comment]
Well, that is the case, when you use the latest and greatest version of phase4. This example that you provided should work fine when using phase4 2.0.0 or higher. The underlying change of enabling variable replacements by default was enabled in ph-commons 11.0.2.

May I ask what version you are using?

**Reply by JerrySP** [reply]
I have used phase4 -1.4.1

**Reply by phax** [reply]
Okay, then please stay tuned for the upcoming 1.4.4 - version. I will backport the change

**Reply by phax** [reply]
@JerrySP phase4 1.4.4 is available from https://github.com/phax/phase4/releases

**Reply by JerrySP** [reply]
@phax Now i can access the env variables in application.properties. Thank you for the support
